Networking/Networking/Remote/OrdersRemote.swift

```diff
@@ -116,9 +116,9 @@ public class OrdersRemote: Remote {
     ///     - status: New Status to be set.
     ///     - completion: Closure to be executed upon completion.
     ///
-    public func updateOrder(from siteID: Int64, orderID: Int64, statusKey: String, completion: @escaping (Order?, Error?) -> Void) {
+    public func updateOrder(from siteID: Int64, orderID: Int64, statusKey: OrderStatusEnum, completion: @escaping (Order?, Error?) -> Void) {
```

Comment:
nit: similar to the comment above, maybe `statusKey` could be renamed to `status` now that it is an enum?